### ISSUE TITLE
Jekyll 4.x support

### ISSUE THREAD

**Issue by clawfire** [forestryio/jekyll-menus#22]
Jekyll 4.0.2 doesn't like your plugin a lot :/ 

```
bundler: failed to load command: jekyll (/Users/thibault/.rvm/gems/ruby-3.0.2/bin/jekyll)
/Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-menus-0.6.1/lib/jekyll/menus.rb:190:in `_fill_front_matter_menu': wrong number of arguments (given 2, expected 1) (ArgumentError)
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-menus-0.6.1/lib/jekyll/menus.rb:130:in `block in _front_matter_menu'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-menus-0.6.1/lib/jekyll/menus.rb:110:in `each'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-menus-0.6.1/lib/jekyll/menus.rb:110:in `_front_matter_menu'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-menus-0.6.1/lib/jekyll/menus.rb:69:in `block (2 levels) in _page_menus'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-menus-0.6.1/lib/jekyll/menus.rb:68:in `map'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-menus-0.6.1/lib/jekyll/menus.rb:68:in `block in _page_menus'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-menus-0.6.1/lib/jekyll/menus.rb:67:in `each'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-menus-0.6.1/lib/jekyll/menus.rb:67:in `each_with_object'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-menus-0.6.1/lib/jekyll/menus.rb:67:in `_page_menus'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-menus-0.6.1/lib/jekyll/menus.rb:18:in `menus'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-menus-0.6.1/lib/jekyll/menus.rb:26:in `to_liquid_drop'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-menus-0.6.1/lib/jekyll/menus/hook.rb:14:in `block in <top (required)>'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-4.2.0/lib/jekyll/hooks.rb:103:in `block in trigger'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-4.2.0/lib/jekyll/hooks.rb:102:in `each'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-4.2.0/lib/jekyll/hooks.rb:102:in `trigger'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-4.2.0/lib/jekyll/site.rb:208:in `render'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-4.2.0/lib/jekyll/site.rb:80:in `process'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-4.2.0/lib/jekyll/command.rb:28:in `process_site'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-4.2.0/lib/jekyll/commands/build.rb:65:in `build'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-4.2.0/lib/jekyll/commands/build.rb:36:in `process'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-4.2.0/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-4.2.0/lib/jekyll/command.rb:91:in `each'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-4.2.0/lib/jekyll/command.rb:91:in `process_with_graceful_fail'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-4.2.0/lib/jekyll/commands/serve.rb:86:in `block (2 levels) in init_with_program'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/jekyll-4.2.0/exe/jekyll:15:in `<top (required)>'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/bin/jekyll:23:in `load'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/bin/jekyll:23:in `<top (required)>'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/bundler-2.2.23/lib/bundler/cli/exec.rb:63:in `load'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/bundler-2.2.23/lib/bundler/cli/exec.rb:63:in `kernel_load'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/bundler-2.2.23/lib/bundler/cli/exec.rb:28:in `run'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/bundler-2.2.23/lib/bundler/cli.rb:474:in `exec'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/bundler-2.2.23/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/bundler-2.2.23/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/bundler-2.2.23/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/bundler-2.2.23/lib/bundler/cli.rb:30:in `dispatch'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/bundler-2.2.23/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/bundler-2.2.23/lib/bundler/cli.rb:24:in `start'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/bundler-2.2.23/exe/bundle:49:in `block in <top (required)>'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/bundler-2.2.23/lib/bundler/friendly_errors.rb:128:in `with_friendly_errors'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/gems/bundler-2.2.23/exe/bundle:37:in `<top (required)>'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/bin/bundle:23:in `load'
        from /Users/thibault/.rvm/gems/ruby-3.0.2/bin/bundle:23:in `<main>'
```

**Comment by clawfire** [comment]
My bad, seems it's more related to ruby 3.x :/

**Comment by DirtyF** [comment]
Duplicate of #21

**Comment by DirtyF** [comment]
This should be fixed in #23 

Note that for Jekyll 4.0 you need to add `gem "webrick"` to your Gemfile.
